packages/common/src/types/savedCharts.ts

```diff
@@ -600,10 +596,7 @@ export const getEChartsChartTypeFromChartKind = (
         case ChartKind.SCATTER:
             return CartesianSeriesType.SCATTER;
         default:
-            return assertUnreachable(
```

Comment:
why are we removing this?

Reply:
Having this function only take a subset of the enum meant that you couldnt call it anywhere you had a ChartKind, so you have to add a typeguard everywhere you want to call it. Letting it take any ChartKind and returning a reasonable default seems a little cleaner. Especially since it's only ever called by our configuration forms. 

If you think it's better to keep the assert and typeguard it, I can put it back.